code/src/index.css

```diff
@@ -11,3 +11,148 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/*Entire site styling*/
+.site-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+/*Form styling*/
+.form-container {
+  background-color: rgb(243, 242, 242);
+  border: solid 1px black;
+  width: 300px;
+  padding-top: 10px;
+  margin-top: 15px;
+  box-shadow: 10px 10px;
+}
+
+.thought-form {
+  display: flex;
+  flex-direction: column;
+}
+
+.form-text {
+  font-size: 14px;
+  margin: 10px;
+}
+
+.input-bar {
+  margin: 10px;
+  padding: 10px;
+  width: 250px;
+}
+
+.send-button {
+  border: none;
+  background: rgb(248, 205, 213);
+  margin: 10px;
+  border-radius: 20px;
+  width: 70%;
+}
+
+.send-heart-emoji {
+  margin: 0px 5px 0px 5px;
+}
+
+/*Styling for each displayed message*/
+.message-container {
+  border: solid 1px black;
+  width: 300px;
+  margin: 25px;
+  display: flex;
+  flex-direction: column;
+  box-shadow: 10px 10px;
+}
+
+.happy-thought {
+  margin: 10px;
+}
+
+.hearts-and-time {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.heart-and-counter {
+  display: flex;
+}
```

Comment:
<img width="80" alt="Screenshot 2021-03-30 at 12 43 36" src="https://user-images.githubusercontent.com/50805742/112977302-2c37c700-9156-11eb-8787-5538c65c3c7c.png">

The heart seems a bit higher that the "x 1"